### PR TITLE
fix: pull quotes markup

### PR DESCRIPTION
**Pull request by tuncaulubilge**
A couple of features were missing in the pull-quotes. 

Added features:
- Support markups inside pull-quotes
- Add "text" field that's used in the ast for extra caption text
- Added section colouring to the pull-quote quotation marks and caption

Also fixed the keys inside article body to clear up the warnings

screen shot 2018-11-16 at 16 13 11
screenshot_1542385151

### CODE REVIEW

File: packages/pull-quote/src/pull-quote.js [hunk 1]
```diff
@@ -2,8 +2,8 @@ import React from "react";
 import PullQuoteBase from "./pull-quote.base";
 import { propTypes, defaultProps } from "./pull-quote-prop-types";
 
-const PullQuote = ({ caption, ...props }) => (
-  <PullQuoteBase {...props}>{caption}</PullQuoteBase>
+const PullQuote = ({ children, ...props }) => (
```
[the remark this answers is not in the document]
Comment (tuncaulubilge) [reply packages/pull-quote/src/pull-quote.js]: Good point, and in that case we don't even need PullQuote. Updated it to export PullQuoteBase directly

[the remark this answers is not in the document]
Comment (tuncaulubilge) [reply packages/pull-quote/src/pull-quote.js]: See c8f7827

**Comment by jimhunty** [comment]
@tuncaulubilge Is the Pull Quote size correct?

**Comment by tuncaulubilge** [comment]
> @tuncaulubilge Is the Pull Quote "quote" size correct?

I haven't changed the quote mark sizes, so it should be same

File: packages/article/__tests__/web/__snapshots__/article-with-style.web.test.js.snap [hunk 2]
```diff
@@ -387,7 +387,9 @@ exports[`full article with style 1`] = `
           <div
             className="c10 S1"
           >
-            <PullQuote />
+            <PullQuote>
```
Comment (colincclark) [inline packages/article/__tests__/web/__snapshots__/article-with-style.web.test.js.snap]: Shouldn't this be mocked outside of the `PullQuote` package?

Comment (tuncaulubilge) [reply packages/article/__tests__/web/__snapshots__/article-with-style.web.test.js.snap]: What do you mean?

Comment (colincclark) [reply packages/article/__tests__/web/__snapshots__/article-with-style.web.test.js.snap]: As discussed, this is all fine

**Pull request merged**